### PR TITLE
What's a CanvasKit?

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-14068
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/me2vj25) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fme2vj25)
teacher: [Link](https://rebrand.ly/ijlropi) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fijlropi)
parent: [Link](https://rebrand.ly/16o0267) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F16o0267)

File: Student/Student/CanvasAppDelegate.swift [hunk 1]
```diff
@@ -88,34 +87,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDelegate {
...
-            token: session.accessToken ?? "",
-            refreshToken: session.refreshToken,
-            clientID: session.clientID,
-            clientSecret: session.clientSecret
-        )!
-        legacyClient.actAsUserID = session.actAsUserID
-        legacyClient.originalIDOfMasqueradingUser = session.originalUserID
-        legacyClient.originalBaseURL = session.originalBaseURL
         let getProfile = GetUserProfileRequest(userID: "self")
         environment.api.makeRequest(getProfile) { response, urlResponse, _ in
```
Comment (inst-danger) [inline Student/Student/CanvasAppDelegate.swift]:
```suggestion
        environment.api.makeRequest(getProfile) { _, urlResponse, _ in
```

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024343&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student, Teacher

[MBL-14068](https://instructure.atlassian.net/browse/MBL-14068)

<details><summary>:x: Swift lint</summary>


    ⚠️  /Users/vagrant/git/Student/Student/CanvasAppDelegate.swift:91:51: Unused Closure Parameter Violation: Unused parameter "response" in a closure should be replaced with _. (unused_closure_parameter)
    Linting 'UIViewControllerExtensions.swift' (42/140)
    Linting 'UIImageExtensions.swift' (43/140)
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024346&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1024347&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|13
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Student/Student/CanvasAppDelegate.swift [hunk 2]
```diff
@@ -88,34 +87,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDelegate {
...
-            legacyUser?.name = session.userName
-            legacyUser?.sortableName = session.userName
-            legacyUser?.shortName = session.userName
-            legacyUser?.avatarURL = response?.avatar_url?.rawValue ?? session.userAvatarURL
-            legacyUser?.loginID = response?.login_id
-            legacyUser?.email = response?.primary_email ?? session.userEmail
-            legacyUser?.calendar = response?.calendar?.ics
-            legacyUser?.sisUserID = ""
-            legacyClient.setValue(legacyUser, forKey: "currentUser")
-            CKIClient.current = legacyClient
```
Comment (armstrongnate) [inline Student/Student/CanvasAppDelegate.swift]: Praise!

**Pull request merged**